### PR TITLE
Adding documentation for reproducing results of the SISC manuscript

### PR DESCRIPTION
**Pull request by fabienevrard**

### CODE REVIEW

File: docs/markdown/paraboloid-polytope_intersection.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Reproducing published results on paraboloid-polytope intersection
+
+This document explains how to reproduce the results presented in our journal paper on paraboloid-polytope intersection, published in the SIAM Journal on Scientific Computing [[link](https://www.siam.org/publications/journals/siam-journal-on-scientific-computing-sisc)]. 
+
+A preprint of this article is available in [arXiv](https://arxiv.org/pdf/2210.07772.pdf) as well as in the [current repository](manuscript_paraboloid_intersection.pdf).
+
+## Unit cube translation (Section 7.1)
+
+In order to reproduce Figures 5 and 6 of the [paper](), it is necessary to build the unit tests of IRL (for which the option `-D BUILD_TESTING=ON` must be provided when building IRL itself). This requires executing the additional command:
```
Comment (robert-chiodi) [inline docs/markdown/paraboloid-polytope_intersection.md:9]: Might be better to say "when configuring IRL itself via CMake".

File: irl/c_interface/paraboloid_reconstruction/c_paraboloid.cpp [hunk 2]
```diff
@@ -195,13 +195,18 @@ void c_Paraboloid_setParaboloidJibben(c_Paraboloid* a_self,
       polygons[n].calculateAndSetPlaneOfExistence();
     }
 
+    double max_vfrac = 0.0;
```
Comment (robert-chiodi) [inline irl/c_interface/paraboloid_reconstruction/c_paraboloid.cpp:198]: Is this related to the reproducibility or is it a separate change? (e.g., do you want it in this Pr or a different one?)

File: docs/markdown/paraboloid-polytope_intersection.md (hunk 1 above)
Comment (fabienevrard) [reply docs/markdown/paraboloid-polytope_intersection.md:9]: Sounds good

File: irl/c_interface/paraboloid_reconstruction/c_paraboloid.cpp (hunk 2 above)
Comment (fabienevrard) [reply irl/c_interface/paraboloid_reconstruction/c_paraboloid.cpp:198]: No that's separate, but a very minor change. Do you want it put aside?

**Pull request closed without merging**